examples/basic-css/pages/index.tsx

```diff
@@ -1,9 +1,11 @@
 import styles from '../styles.module.css'
 
-export default function Home() {
+const Home = () => {
```

Comment:
Would be good to keep here as before with the function name, that's also helpful for react devtools and align with react docs

Reply:
I agree. I considered this my stylistic preference, but if this is what the React docs do as well, I think I would prefer `function` over `const` as well, so the declaration and `export` aren't split